cmd/lncli/invoicesrpc_active.go

```diff
@@ -212,12 +217,14 @@ func addHoldInvoice(ctx *cli.Context) error {
 
 	args = args.Tail()
 
-	switch {
-	case ctx.IsSet("amt"):
+	if ctx.IsSet("amt") {
 		amt = ctx.Int64("amt")
-	case args.Present():
+	}
+	if ctx.IsSet("amt_msat") {
```

Comment:
Nit: the `IsSet` checks aren't needed here as if they're not set, then they'll just be 0 (the default).

Reply:
Will remove it.